xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/renameStatus.vm

```diff
@@ -46,9 +46,13 @@
     'renameId': $renameJobRequest.id.get($mathtool.sub($renameJobRequest.id.size(), 1))
   })))
   <div class="xcontent job-status" data-url="$jobStatusURL">
-    <h2>$services.localization.render('core.rename.status.label')</h2>
-    <p class="text-muted small">$services.localization.render('core.rename.status.hint',
-      [$xwiki.getUserName($renameJobRequest.userReference), $xwiki.formatDate($renameJobStatus.startDate)])</p>
+    <div class="row document-header">
```

Comment:
you should use the macro in all the needed instances. Also on create.vm

Reply:
I had plan to roll it back today.
You said that the templates should remain as before since bootstrap is not necessary applicable for them.
Should I roll it back?

Reply:
yes, you are right, if you created a copy, these should be rolledback.